### ISSUE TITLE
KRouterLink should be outlined on focus, by default

### ISSUE THREAD

**Issue by jonboiser** [learningequality/kolibri-design-system#130]
In https://github.com/learningequality/kolibri/pull/7660, the missing focus outlines are fixed by explicitly adding

```
:class="$computedClass({ ':focus': $coreOutline })"
```

to some `KRouterLink`s, anchor tags, and `router-links` to achieve that focus outline. 

This fixes the original issue in a few places, but not globally, and isn't scalable. So I think this styling needs to be applied to link-like things as a default.

**Comment by indirectlylit** [comment]
see also:

https://github.com/learningequality/kolibri-design-system/issues/131 (All components, including KTextBox, should have a standard focus outline)

**Comment by sairina** [comment]
Closing this issue, as the K* components are working as expected.
